Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_cff.py

```diff
@@ -7,6 +7,10 @@
 import EventFilter.HcalRawToDigi.HcalRawToDigi_cfi
 hcalDigiAlCaMB = EventFilter.HcalRawToDigi.HcalRawToDigi_cfi.hcalDigis.clone()
 import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi
+hbherecoMBspecial = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi.hbheprereco.clone()
```

Comment:
@bsunanda - would it be easy to change "special" to something more physically meaningful? (for future code readability)